rudder-agent/SOURCES/rudder-agent-postinst

```diff
@@ -50,76 +42,75 @@ if type slibclean >/dev/null 2>&1; then
+  cp -r ${CFE_DIR}/inputs/rudder-server-roles.conf /var/backups/rudder/
   RESTORE_SERVER_ROLES_BACKUP=1
 fi
 
 # Copy initial promises if there aren't any already or,
 # if the cf-promises validation fails, it means we have a broken set of promises (possibly a pre-2.8 set).
 # Reset the initial promises so the server is able to send the agent a new set of correct ones.
 RUDDER_UUID=`cat /opt/rudder/etc/uuid.hive 2>/dev/null || true`
-if [ ! -f /var/rudder/cfengine-community/inputs/promises.cf ] || ! /var/rudder/cfengine-community/bin/cf-promises >/dev/null 2>&1 && [ "${RUDDER_UUID}" != "root" ]
+if [ ! -f ${CFE_DIR}/inputs/promises.cf ] || ! ${CFE_DIR}/bin/cf-promises >/dev/null 2>&1 && [ "${RUDDER_UUID}" != "root" ]
 then
-  rm -rf /var/rudder/cfengine-community/inputs/* || true
-  cp -r /opt/rudder/share/initial-promises/* /var/rudder/cfengine-community/inputs/
+  rm -rf ${CFE_DIR}/inputs/* || true
+  cp -r /opt/rudder/share/initial-promises/* ${CFE_DIR}/inputs/
 fi
 
 # Restore rudder-server-roles.conf if necessary
 if [ "${RESTORE_SERVER_ROLES_BACKUP}" = "1" ]; then
-  cp -r /var/backups/rudder/rudder-server-roles.conf /var/rudder/cfengine-community/inputs/rudder-server-roles.conf
+  cp -r /var/backups/rudder/rudder-server-roles.conf ${CFE_DIR}/inputs/rudder-server-roles.conf
 fi
 
 # Create a key if we don't have one yet
-if [ ! -f /var/rudder/cfengine-community/ppkeys/localhost.priv ]
+if [ ! -f ${CFE_DIR}/ppkeys/localhost.priv ]
 then
   echo -n "INFO: Creating keys for CFEngine agent..."
-  /var/rudder/cfengine-community/bin/cf-key >> /var/log/rudder/install/rudder-agent.log 2>&1
+  ${CFE_DIR}/bin/cf-key >> /var/log/rudder/install/rudder-agent.log 2>&1
   echo " Done."
 fi
 
+# Remove cfengine lock log file : http://www.rudder-project.org/redmine/issues/5488
+rm -f ${CFE_DIR}/cf3.*.runlog*
+
 # Remove the lock on CFEngine
 if [ ${I_SET_THE_LOCK} -eq 1 ]; then
-  rm -f /opt/rudder/etc/disable-agent
+  ${RUDDER_CMD} agent enable
 fi
 
-# Remove cfengine lock log file : http://www.rudder-project.org/redmine/issues/5488
-rm -f /var/rudder/cfengine-community/cf3.*.runlog*
-
 # Restart daemons if we stopped them, otherwise not
 if [ ${CFRUDDER_FIRST_INSTALL} -ne 1 ]
 then
   # Check if agent is disabled
   if [ ! -f /opt/rudder/etc/disable-agent ]
   then
-    if [ -r /var/rudder/cfengine-community/inputs/failsafe.cf -o -r /var/rudder/cfengine-community/inputs/promises.cf ]
+    if [ -r ${CFE_DIR}/inputs/failsafe.cf -o -r ${CFE_DIR}/inputs/promises.cf ]
     then
-      ${AGENT_START_CMD} || true
+      ${RUDDER_CMD} agent start || true
     fi
   else
     echo "********************************************************************************"
     echo "rudder-agent has been updated, but was not started as it is disabled."
-    echo "To enable rudder agent, you have to remove disable file, and start rudder-agent:"
-    echo "# rm -f /opt/rudder/etc/disable-agent"
-    echo "# ${AGENT_START_CMD}"
+    echo "To enable rudder agent:"
```

Comment:
I think this should say "To enable and start Rudder agent" (or "To enable and start rudder-agent")